docs/en/observability/synthetics-lightweight.asciidoc

```diff
@@ -0,0 +1,34 @@
+[[synthetics-lightweight]]
+= Advanced lightweight monitors
+
+// not sure what should be included here, but this is from existing
+// docs: https://www.elastic.co/guide/en/observability/current/uptime-set-up-choose-project-monitors.html#uptime-set-up-project-monitors-add
```

Comment:
Thanks! This is helpful. I just pushed changes to this page. I did this in two commits for you to compare what's from the Heartbeat docs vs. my edits. I copied over from Heartbeat in https://github.com/elastic/observability-docs/pull/2353/commits/16cc026a233624fad0b73b04931dcf6b1eed9e04, then I edited it down in https://github.com/elastic/observability-docs/pull/2353/commits/a14999fb18e7f37ddb18884e6b693afd83ff9e4c. 

The preview will be [here](https://observability-docs_2353.docs-preview.app.elstc.co/guide/en/observability/master/synthetics-lightweight.html) after the build finishes. Let me know what you think!

Reply:
Much nicer! I was actually thinking we could eventually copy over the individual pages for `HTTP` `TCP` and `ICMP`. 

If you're using a project based lightweight monitor it doesn't make sense to have to go to the heartbeat docs, ideally you don't even know what heartbeat is.

If it's too much for this PR I understand, but I think it makes sense to do before GA. Your thoughts?

Reply:
>If it's too much for this PR I understand, but I think it makes sense to do before GA. Your thoughts?

I totally agree, but I think we should do it in a follow-up PR. I'll open an issue to track this. We might even be able to get this in before 8.6 if we want to, but I think it would be overwhelming in this PR.

Reply:
https://github.com/elastic/observability-docs/issues/2372